uw-frame-components/js/app-config.js

```diff
@@ -11,10 +11,10 @@ define(['angular'], function(angular) {
         })
         .value('SERVICE_LOC', {
             'aboutURL': null,
-            'groupURL': '/portal/api/groups',
+            'groupURL': 'staticFeeds/groups.json',
```

Comment:
Concern: is this the default value as experienced by frame-based apps? Such that frame-based apps not overriding this value will stop correctly grabbing groups from `/portal/api/groups` unless they start overriding this value in app-local config?